system/datatypes/peer_reviewed_article.php

```diff
@@ -1099,21 +1123,14 @@ protected function createDiscussionsDomElement($type) {
 				if (!$review && (bool)$comment->getAttribute(self::FIELD_NAME_DISCUSSION_COMMENT_PENDING)) {
 					continue;
 				}
-				$createdAt = date('j-m-y, H:i', ltrim($comment->getAttribute(self::FIELD_NAME_CREATED_AT), 't'));
-				$committerName = $this->getCommentCommenter($comment);
-				$commentHtml = nl2br(htmlspecialchars($comment->getText()));
-				$commentHtml .= '<p>' . __('art-by') . ' <strong>' . htmlspecialchars($committerName) . '</strong> ' . __('art-at') . ' ' . htmlspecialchars($createdAt);
-				if (!$review && isUser()) {
-					$committerEmail = $comment->getAttribute(self::FIELD_NAME_DISCUSSION_COMMENTER_EMAIL);
-					$commentHtml .= ' <span> | ' . __('art-email') . ': <a href="mailto:' . htmlspecialchars($committerEmail) . '">' . htmlspecialchars($committerEmail) . '</a></span>';
-				}
+
+				$li = $this->html->createElement('li')->appendTo($commentList);
+				$commentHtml = $this->createCommentDomElement($comment, $review, $firstComment && $blocking);
+				$li->append($commentHtml);
+
 				if ($firstComment) {
-					if ($blocking) {
-						$commentHtml .= ' | ' . ($resolved ? __('art-is-resolved') : __('art-is-blocking'));
-					}
+					$li->addClass("first-comment");
```

Comment:
It would be more diverse if the class would be separated so that all comments get the "`comment`" class and the first comment would get "`first`" as well.

Reply:
Maybe, or maybe using `:first-child` could remove the need for this class entirely. However, this just moves the existing class, so I'm going to consider this out of scope for the PR for now.

Reply:
agreed